### ISSUE TITLE
[REQ] Export results to plist (with proper blob support?)

### ISSUE THREAD

**Issue by sequelpro** [sequelpro/sequelpro#960]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=960
Reporter:gco...@gmail.com Date:2011-01-24 09:31:47 Status:New

We've got an export to XML, but I'd like to export to a property list (plist) file. 

Standard number and string plist types would go a long ways, but it would be a super-super-huge plus if the export properly handled BLOBs and hexcoded them as NSData &lt;data&gt; blocks. 

Perhaps someone might have mentioned something like this elsewhere, but in the event of NULL or empty strings, just leave the entry blank in the plist, rather than including &quot;(null)&quot;  like MySQL Query Browser does.

---

Basically, what I'm looking for is something that would allow me to export a set of results directly to a property list, and have that plist directly readable in Objective-C, without having to muck around with it first.

**Comment by fulldecent** [comment]
I recommend supporting JSON before PLIST.
